src/control/server/engine/exec_test.go

```diff
@@ -119,13 +120,18 @@ func TestRunnerNormalExit(t *testing.T) {
 
 	// set this to control the behavior in TestMain()
 	os.Setenv(testModeVar, "RunnerNormalExit")
-	// verify that user env gets overridden by config
-	os.Setenv("OFI_INTERFACE", "bob0")
```

Comment:
Seems like this case could still be valid if the user allows passthrough for a variable that's also in the config. I think it's worth keeping that behavior defined in the tests.